neo/rawio/tdtrawio.py

```diff
@@ -308,13 +310,16 @@ def _parse_header(self):
                 units = "uV"  # see https://github.com/NeuralEnsemble/python-neo/issues/1369
                 gain = 1.0
                 offset = 0.0
-                signal_channels.append((chan_name, str(chan_id), sampling_rate, dtype, units, gain, offset, stream_id))
+                buffer_id = ""
+                signal_channels.append((chan_name, str(chan_id), sampling_rate, dtype, units, gain, offset, stream_id, buffer_id))
 
         if missing_sev_channels:
             warnings.warn(f"Could not identify sev files for channels {missing_sev_channels}.")
 
         signal_streams = np.array(signal_streams, dtype=_signal_stream_dtype)
         signal_channels = np.array(signal_channels, dtype=_signal_channel_dtype)
+        # buffer concept here, data are spread per channel and paquet
```

Comment:
Qu'est-ce que c'est un paquet la? channel are files and the paquet is for folder? or packets of info meaning like internal blocks of data like data packets?

Reply:
Yes this is what I meant